recipes/julia/julia-wrapper.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+JULIA_HOME="$(dirname "${BASH_SOURCE[0]}")"
+
+# Set JULIA_PKGDIR to $PREFIX/share/julia/site to avoid contaminating
+# user's ~/.julia/.  This script will modify Pkg.dir() and
+# Base.LOAD_CACHE_PATH[1] appropriately.  Note that wrapper script is
+# used rather than /etc/julia/juliarc.jl because LOAD_CACHE_PATH is
```

Comment:
Though juliarc is going to be in the user's home directory, right? It would be better if we had a way to set this *without* touching the user's home directory. Is there one?